examples/src/test/scala/akka/projection/examples/KafkaToSlickIntegrationSpec.scala

```diff
@@ -0,0 +1,184 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.projection.examples
```

Comment:
Good if this is placed in a non-akka package, `docs.projection.exampes` or such. Then we verify that we don't use any internal APIs here.

Reply:
Ah, my example uses `SlickOffsetStore` to assert offset state.  I was thinking this example could serve also as an integration test, but maybe we need both. Should we create an `integration` project that can remain in the `akka` namespace and then clean up the example to not reference internal APIs?

Reply:
Keep it as is. We'll see when we write the doc examples.

Reply:
May I suggest the Alpakka standard of `docs.scaladsl` and `docs.javadsl`.